JUnit version 4.12
I...E
Time: 1.592
There was 1 failure:
1) test3(org.joda.time.chrono.ISOChronology_ESTest)
org.joda.time.IllegalFieldValueException: Value 1612 for minuteOfHour must be in the range [0,59]
	at org.joda.time.field.FieldUtils.verifyValueBounds(FieldUtils.java:218)
	at org.joda.time.field.PreciseDateTimeField.set(PreciseDateTimeField.java:111)
	at org.joda.time.chrono.BaseChronology.getDateTimeMillis(BaseChronology.java:164)
	at org.joda.time.chrono.AssembledChronology.getDateTimeMillis(AssembledChronology.java:151)
	at org.joda.time.chrono.AssembledChronology.getDateTimeMillis(AssembledChronology.java:148)
	at org.joda.time.chrono.ZonedChronology.getDateTimeMillis(ZonedChronology.java:129)
	at org.joda.time.chrono.AssembledChronology.getDateTimeMillis(AssembledChronology.java:148)
	at org.joda.time.chrono.ISOChronology_ESTest.test3(ISOChronology_ESTest.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)

FAILURES!!!
Tests run: 3,  Failures: 1

